common/src/main/java/com/wynntils/core/functions/FunctionManager.java

```diff
@@ -119,21 +120,36 @@ public Component getSimpleValueString(
             return header.append(Component.literal("??"));
         }
 
-        String formattedValue = format(value.get());
+        String formattedValue = format(value.get(), false, 2);
 
         return header.append(Component.literal(formattedValue).withStyle(color));
     }
 
-    public String getRawValueString(Function<?> function, FunctionArguments arguments) {
+    public String getRawValueString(
+            Function<?> function, FunctionArguments arguments, boolean formatted, int decimals) {
         Optional<Object> value = getFunctionValueSafely(function, arguments);
         if (value.isEmpty()) {
             return "??";
         }
 
-        return format(value.get());
+        return format(value.get(), formatted, decimals);
     }
 
-    private String format(Object value) {
+    private String format(Object value, boolean formatted, int decimals) {
+        if (value instanceof Number number) {
+            if (formatted) {
+                // French locale has NBSP
```

Comment:
Swedish locale, too. And a decimal comma instead of a decimal point. Overall, I think we need to think a bit whether we want to support locale formatting here, or just use `Locale.ROOT`. Maybe it should even be a setting.

Reply:
Yes. I thought about that. Or only using US locale or smth.. Not sure atm.